CMakeLists.txt

```diff
@@ -113,7 +113,7 @@ set(NVIM_VERSION_PATCH 2)
 set(NVIM_VERSION_PRERELEASE "-dev") # for package maintainers
 
 # API level
-set(NVIM_API_LEVEL 5)         # Bump this after any API change.
+set(NVIM_API_LEVEL 6)         # Bump this after any API change.
```

Comment:
No need to bump this. One can always just use the latest level in a PR; if it needs to be bumped, CI  (or `make functionaltest` locally) will tell you.